Packs/Incapsula/Scripts/IncapWhitelistCompliance/IncapWhitelistCompliance.py

```diff
@@ -0,0 +1,106 @@
+import re
+
+from CommonServerPython import *  # noqa: F401
+
+import demistomock as demisto  # noqa: F401
+
+# Consts
+COMPLIANCE_LABEL = "Compliance"
+CURL_GET_COMMAND = 'curl --insecure -L -I -m 1 http://{0} | grep -C 1 "200 OK"'
+HTTP_OK = "200 OK"
+MAIL_TO_RECIPIENT = "IncapsulaUpdates@demisto.com"
+INCAPSULA_WARNING = "****Incapsula Warning****"
+BASIC_WARNING_MAIL = '''
+WARNING
+Your Site {0} is not compliant to the Incapsula allow list policy
+please update your settings
+
+--DBot
+'''
+ADVANCE_WARNING_MAIL = '''
```

Comment:
Not used.
See comment in the `escalation` method.

Reply:
removed